aws/ae-environment.yml

```diff
@@ -97,6 +97,3 @@ Resources:
         AvailabilityZones:
           Fn::GetAZs:
             Ref: "AWS::Region"
-
-    EpochElasticIP:
```

Comment:
Would removing this section stop billing of the ElasticIP? Do I understand correctly that manual operation is needed for actually deleting ElasticIP? Did you plan for those operations?

Reply:
Already deleted.